examples/ruby/ruby_client.rb

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env ruby
+
+# Load the gem
```

Comment:
We may want to make some sort of assertion on what ruby versions are allowed near here